### PR TITLE
New Security Hardsuit, "Patrol".

### PR DESCRIPTION
**Pull request by No-Dad-Not-The-Belt**
## About the PR
_All sector security personnel, report to the armory for re-arm and resupply._

This PR aims to add a new security hardsuit, the patrol variant. Sporting a fresh updated look, this new suit uses the same protection values as the original while only effecting your sprint and walk speed modifier by 15%. This should help combat the speed difference between security and...***Cough Cough*** "frontier personnel".

## DISCLAMER
I am in no way a sprite artist, and this may only be my fourth or fifth time making sprites. Any feedback/criticism is welcome and encouraged.

**Media**

Security Patrol Hardsuit

New Sec Icon

icon

**Changelog**

:cl:
- add: Added the Security Patrol Hardsuit

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 1]
```diff
@@ -187,6 +187,36 @@
...
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Hardsuits/security-patrol.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Hardsuits/security-patrol.rsi
+  - type: PressureProtection
+    highPressureMultiplier: 0.525
+    lowPressureMultiplier: 10000
+  - type: ClothingSpeedModifier
+    walkModifier: 0.85
+    sprintModifier: 0.85
```
Comment (dvir001) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:207]: sprintModifier: 0.80 ~
To be a middle point from sec to brig suit

Comment (No-Dad-Not-The-Belt) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:207]: Dig it. Will send another commit with the correction.

**Comment by Kesiath** [comment]
that goes hard and looks turbo evil, I'd suggest recoloring the helmet light to be more in-line with the other security lights though

**Comment by Cheackraze** [comment]
yea i like it but i also agree the dark red visor makes it very very evil, if we could lighten it or yea just match more sec colored light it would be great, otherwise that looks like a nukie hunter killer or something

**Comment by No-Dad-Not-The-Belt** [comment]
> that goes hard and looks turbo evil, I'd suggest recoloring the helmet light to be more in-line with the other security lights though - also I'd make it a bit less protective then standard sec armor if it's meant to be the 'lighter armor' variant. Should be a choice for officers to make, if they want to move faster or have higher protection.

Fair, ill change the protections around a bit. I like the idea of it being kind of a choice for Sec Offs to decide if they want to be more mobile or better protected. As for how to get it in game do you think it should have a crafting recipe at the Sec Fab/ "removing" material from a normal sec suit or just leave it up to mappers to generate them?

**Comment by No-Dad-Not-The-Belt** [comment]
> yea i like it but i also agree the dark red visor makes it very very evil, if we could lighten it or yea just match more sec colored light it would be great, otherwise that looks like a nukie hunter killer or something

Yeah i was kinda going for a more indimidating style as in my head cannon this would be the suit sec officers doing traffic stops and boarding parties would be wearing. How would you feel about lightening the color pallet on the visor a little but but keeping it red, and adding a helmet attached torch for lighting using the regular yellow #FFEEAD color for light? Examples listed in the picture below for inspo. 

Example helmet light


Example helmet light 2

**Comment by Kesiath** [comment]
> How would you feel about lightening the color pallet on the visor a little but but keeping it red, and adding a helmet attached torch for lighting using the regular yellow #FFEEAD color for light?

Headlamp is a nice idea, ERT suits make use of the same concept. Keeping a red visor is fine, but then maybe we'd want to repass the other suits for consistency sake and do something similar? Default sec armor kinda sucks as is anyways. The only criticism I have left with it now is that the incredibly dark colors and aggressive design makes it look more like deathsquad armor.

**Comment by No-Dad-Not-The-Belt** [comment]
> 

To be honest the helmet in use for this hardsuit is just the default sec hardsuit helmet recolored, so I could easily go through and change the visors to make it more consistent, and add a headlamp to them as well. Like I said before I do really like the aggressive style of it, but thinking about it again maybe this sprite would be better suited for the more armored variant (the regular sec hardsuit) and use the old sec armor for the patrol armor? Dont know. Ill do some mock ups of the new helmet redesign tomorrow and upload some pictures of it here though.

**Comment by No-Dad-Not-The-Belt** [comment]
> Headlamp is a nice idea, ERT suits make use of the same concept. Keeping a red visor is fine, but then maybe we'd want to repass the other suits for consistency sake and do something similar? Default sec armor kinda sucks as is anyways. The only criticism I have left with it now is that the incredibly dark colors and aggressive design makes it look more like deathsquad armor.

How would something like this be? Images below are the icon and flash icon image as i dont want to fully sprite it until i get a "looks good".

New Sec Helmet

New Sec Helmet - Flash

**Comment by Kesiath** [comment]
I think that looks fairly neat, would wanna see in game but it reminds me of the default EVA hardsuit helmets, and those are fine.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**